example/index.tsx

```diff
@@ -212,7 +231,9 @@ const App = () => {
 const DynamicReader: React.FC = () => {
   const { manifestUrl } = useParams<{ manifestUrl: string }>();
   const decoded = decodeURIComponent(manifestUrl);
-  return <WebReader webpubManifestUrl={decoded} />;
+  return (
+    <WebReader injectables={htmlInjectables} webpubManifestUrl={decoded} />
```

Comment:
@kristojorg Thanks for this! I'm not totally clear on how these will get passed to the `useHtmlReader` though, as I can't see the `...props` getting passed. Am I missing something, or should these be somehow passed to https://github.com/NYPL-Simplified/web-reader/blob/main/src/useWebReader.tsx#L55?

Reply:
Nope you are entirely correct, I was seeing it working in my npm linked project due to browser cacheing. Thanks for catching this! I should really write some tests for this functionality.